benchmark/cajita/CMakeLists.txt

```diff
@@ -22,14 +22,11 @@ if(Kokkos_ENABLE_SERIAL OR Kokkos_ENABLE_CUDA OR Kokkos_ENABLE_OPENMP OR Kokkos_
     endif()
 
   if(Cabana_ENABLE_TESTING)
```

Comment:
By the way why are we guarding with ` if(Cabana_ENABLE_TESTING)`?

Reply:
You mean we should use `BUILD_TESTING` instead?

Reply:
I wouldn't put any condition (so yes rely on `BUILD_TESTING`)

Reply:
I usually either want to build the benchmark only or build tests for CI. And we only enable tests with `Cabana_ENABLE_TESTING` on